docs/ref/models/fields.txt

```diff
@@ -1424,7 +1424,8 @@ The possible values for :attr:`~ForeignKey.on_delete` are found in
 .. attribute:: ForeignKey.to_field
 
     The field on the related object that the relation is to. By default, Django
-    uses the primary key of the related object.
+    uses the primary key of the related object, therefore ensure the referenced
```

Comment:
I see, but the idea is to keep clear that if your not using the default from Django (referencing to a pk field), you might then use a field that is `unique=True`.
So I think I'd put as `therefore be sure you're referencing a field that is unique=True in order to stay aligned with the design`.